src/mimefactory.rs

```diff
@@ -454,7 +454,7 @@ impl MimeFactory {
                 };
                 stock_str::subject_for_new_contact(context, self_name).await
             }
-            Loaded::Mdn { .. } => stock_str::read_rcpt(context).await,
+            Loaded::Mdn { .. } => "Read Receipt".to_string(), // untranslated to no reveal sender's language
```

Comment:
Maybe "Receipt notification" is better as an MDNs is auto-sent and the message may not be read yet

Reply:
makes sense